onebusaway-gtfs/src/main/java/org/onebusaway/gtfs/model/StopTime.java

```diff
@@ -97,6 +97,10 @@
   @CsvField(optional = true)
   private String track;
 
+  // Custom extension for MNR
```

Comment:
Let's try not to name agencies, this module tries to be generic.  Certainly not a blocker but a preference.

Reply:
Thanks there were a few other comments in the Trip class that also called out a similar custom extension. Would you like me to clean this up and those comments up in a future pull request?